app/redux/ducks/translations.js

```diff
@@ -14,9 +14,9 @@ const initialState = {
   locale: DEFAULT_LOCALE,
   strings: {
     project: {},
-    workflow: {
-      tasks: {}
```

Comment:
I was initially worried that removing that initial `task: {}` statement here might affect other esoteric parts of the PFE Classifier that blindly calls `workflow.tasks[taskName]` without checking for existence... but, after re-reading the code, I realise that probably isn't happening anywhere yet, and I'm confusing `translations.string.workflow` with _other_ `this.props.workflow` definitions.

Still, reminder to self: test this assumption.

Reply:
I think this should be ok. `workflow.tasks` was removed in #4199 but I missed this line.